easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018a-Python-2.7.14-HDF5-1.10.1.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.3'
+hdf5_ver = '1.10.1'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s' % hdf5_ver
```

Comment:
no need to customize this for `HDF5` 1.10.1 (but you still need a `GDAL` easyconfig for `Python` 2.7.14, that's not there yet)